packages/styling/src/interfaces/IPalette.ts

```diff
@@ -242,4 +242,24 @@ export interface IPalette {
    * Color code for greenLight.
    */
   greenLight: string;
+
+  /**
```

Comment:
We should ONLY be using system colors for high contrast modes:
https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#System_Colors
@betrue-final-final